src/topology/continuous_function/bounded.lean

```diff
@@ -1030,4 +1031,101 @@ module over the algebra of bounded continuous functions from `α` to `𝕜`. -/
 
 end normed_algebra
 
+section normed_lattice_ordered_group
+
+variables [topological_space α] [normed_lattice_add_comm_group β]
+
+-- Can we infer this from https://github.com/leanprover-community/mathlib/blob/f29b0b49badab1bcbe338cb79d102e36e79dce09/src/topology/continuous_function/basic.lean#L114 ?
+instance : partial_order (α →ᵇ β) := partial_order.lift (λ f, f.to_fun) (by tidy)
+
+/--
+Continuous normed lattice group valued functions form a meet-semilattice
+-/
+instance : semilattice_inf (α →ᵇ β) :=
+{ inf := λf g,
+  { to_fun := λ t, f(t)⊓g(t),
+    continuous_to_fun := begin
+      continuity,
+    end,
+    bounded' := begin
+      cases f.bounded' with C₁ hf,
+      cases g.bounded' with C₂ hg,
+      use C₁+C₂,
+      intros,
+      simp,
```

Comment:
This `simp` is non terminal: it does not close the goal. The problem with this is that when the set of simp lemmas change, it might simplify the expression to something different and the proof will break. Therefore we don't allow non-terminal simp.
Please use `squeeze_simp` and replace that `simp` with the `simp only [...]` from the suggestion you get.

Reply:
Apparently here you needed only `dsimp only,`

Reply:
@RemyDegenne Thanks for the review. Hopefully I have fixed all these issues now.